### PR TITLE
Focus on input on edit chat title clicked

### LINKED ISSUE #2519: Focus input when edit chat title or prompter message

### PR DESCRIPTION
**Pull request by bishowX**
This PR brings following changes:
1. When clicking on edit chat title button the input field will now be on focus.
2. Clicking on edit prompt button will set the focus on chat box

Fixes #2519

### CODE REVIEW

**Comment by bishowX** [comment]
I'm open to discussions around is this a good user experience or we should do it differetly

File: website/src/components/Chat/ChatForm.tsx [hunk 1]
```diff
@@ -44,6 +44,7 @@ export const ChatForm = forwardRef<HTMLTextAreaElement, ChatFormProps>((props, r
         _dark={{
           bg: "gray.800",
         }}
+        autoFocus
```
Comment (notmd) [inline website/src/components/Chat/ChatForm.tsx:47]: Please remove this, we only want focus the inline input, not this one

File: website/src/components/Chat/ChatListItem.tsx [hunk 2]
```diff
@@ -95,6 +95,10 @@ export const ChatListItem = ({
             pe="50px"
             borderRadius="lg"
             maxLength={100}
+            autoFocus
+            onFocus={(e) => {
+              e.target.select();
+            }}
```
Comment (notmd) [inline website/src/components/Chat/ChatListItem.tsx:101]: I dont think we need to do this, just focus the input is ok

**Review by notmd — COMMENTED** [review]
Thank you

**Comment by bishowX** [comment]
> Also can you upload a screen record here?

Just pushed the changes to focus prompt input. which screen record do you want?

**Review by notmd — APPROVED** [review]
Thank you!

**Pull request merged**